messages/es.json

```diff
@@ -0,0 +1,42 @@
+{
+  "LocaleSwitcher": {
+    "label": "Cambia idioma",
+    "locale": "{locale, select, de {🇩🇪 Deutsch} en {🇺🇸 English} es {🇪🇸 Español} other {Unknown}}"
+  },
+  "Landing": {
+    "welcome": "¡Bienvenidos a MeshForms!"
+  },
+  "JoinForm": {
+    "title": "Únase NYC Mesh",
+    "banner": "",
+    "sections": {
+      "personalInfo": "Información Personal",
+      "addressInfo": "Información de dirección"
+    },
+    "fields": {
+      "firstName": "Nombre",
+      "lastName": "Apellido",
+      "emailAddress": "Dirección de correo electronico",
+      "phoneNumber": {
+        "phoneNumber": "Número de teléfono",
+        "error": ""
+      },
+      "streetAddress": "Dirección de calle",
+      "unit": "Numero de apartamento",
+      "city": "Ciudad",
+      "zipCode": "Codigo postal",
+      "roofAccess": "Marque esto si tiene acceso al techo",
+      "reference": "¿Cómo se enteró de nosotros?",
+      "ncl": "Acepto la Licencia de Red Común",
+      "submit": {
+        "submit": "Entregar",
+        "loading": "Cargando...",
+        "thanks": "¡Gracias!"
+      }
+    },
+    "states": {
+      "NY": "Nueva York",
+      "NJ": "Nueva Jersey"
```

Comment:
Do proper nouns really get translated like this? Wild

Reply:
Sometimes. We could probably do it either way. I want to ask BAM to look over these and also give us translations for Chinese and French Creole.